tests/DI/Compiler.generatedFactory.returnTypes.phpt

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/**
+ * Test: Nette\DI\Compiler: generated services factories from interfaces with return type declarations.
+ * @phpVersion 7.0
+ */
+
+use Nette\DI;
+use Tester\Assert;
+use Nette\DI\Config\Adapters as Adapt;
```

Comment:
A little leftover from when I used another test as a template for this one. Removed. :)